planetary-social/nos
# #1487: Remove media links from notes

## Issues covered
#1487 

## Description
Removes media links from notes. Images and videos, including .png, .jpg, .mp4, .webp, and others.

The ticket says:

> I should see the media
> or a missing / broken link option.

We don't have a missing / broken link placeholder at this point. I don't think we have a ticket for one, either, though I do think we have design for it. I'll create a ticket. Done. #1583

## How to test
1. Navigate to a note with media links, or create your own!
2. You will not see media links.

## Screenshots/Video
https://github.com/user-attachments/assets/0c079ce4-fab9-486c-8633-0d0be2164c37
